folio-spring-base/src/main/java/org/folio/spring/DefaultFolioExecutionContext.java

```diff
@@ -36,8 +36,10 @@ public DefaultFolioExecutionContext(FolioModuleMetadata folioModuleMetadata,
     Map<String, Collection<String>> allHeaders) {
     this.folioModuleMetadata = folioModuleMetadata;
     this.allHeaders = allHeaders;
-    this.okapiHeaders = new HashMap<>(allHeaders);
```

Comment:
Let's simplify this
    this.okapiHeaders = **new TreeMap<>(String.CASE_INSENSITIVE_ORDER)**;
    this.okapiHeaders.putAll(allHeaders);